eventmesh-runtime/src/main/java/org/apache/eventmesh/runtime/util/EventMeshUtil.java

```diff
@@ -295,20 +288,20 @@ public static void printState(final ThreadPoolExecutor scheduledExecutorService)
     @SuppressWarnings("unchecked")
     public static <T> T cloneObject(final T object) throws IOException, ClassNotFoundException {
         try (ByteArrayOutputStream byOut = new ByteArrayOutputStream();
-            ObjectOutputStream outputStream = new ObjectOutputStream(byOut)) {
+             ObjectOutputStream outputStream = new ObjectOutputStream(byOut)) {
 
             outputStream.writeObject(object);
 
             try (ByteArrayInputStream byIn = new ByteArrayInputStream(byOut.toByteArray());
-                ObjectInputStream inputStream = new ObjectInputStream(byIn)) {
+                 ObjectInputStream inputStream = new ObjectInputStream(byIn)) {
                 return (T) inputStream.readObject();
```

Comment:
I'm afraid your CheckStyle plugin was misconfigured. Please have a check.

@yanrongzhen Is it a side effect of Spotless plugin? This contributor once ran `checkStyleMain` gradle task.

Reply:
I think it's not a side effect of Spotless, I tried to perform `checkStyleMain`, but failed to reproduce.